InnerEye/ML/config.py

```diff
@@ -788,9 +793,13 @@ def get_full_image_sample_transforms(self) -> ModelTransformsPerExecutionMode:
         """
         from InnerEye.ML.utils.transforms import Compose3D
         from InnerEye.ML.photometric_normalization import PhotometricNormalization
+        from InnerEye.ML.augmentations.augmentation_for_segmentation_utils import BasicAugmentations
 
         photometric_transformation = Compose3D(transforms=[PhotometricNormalization(self, use_gpu=False)])
-        return ModelTransformsPerExecutionMode(train=photometric_transformation,
+        training_augmentations = Compose3D(
+            transforms=[PhotometricNormalization(self, use_gpu=False), BasicAugmentations()])
+        training_transformation = training_augmentations if self.augmentations else photometric_transformation
```

Comment:
DRY?
transforms= [Photometric()]
if self.augmentations:
    transforms.append(Basic..)
...Compose3D(transforms)

Reply:
for validation and test I dont want all the transforms so I need 2 lists